integration-tests/src/main/java/org/apache/druid/testing/utils/KafkaEventWriter.java

```diff
@@ -42,8 +42,8 @@
   public KafkaEventWriter(IntegrationTestingConfig config, boolean txnEnabled)
   {
     Properties properties = new Properties();
-    KafkaUtil.addPropertiesFromTestConfig(config, properties);
     properties.setProperty("bootstrap.servers", config.getKafkaHost());
+    KafkaUtil.addPropertiesFromTestConfig(config, properties);
```

Comment:
Maybe move this to line 56 so that the config can override other properties like `acks` if needed in the future